### PR TITLE
[ERSSUP-64385]-[TBC]-[Update Overview]-[NS]

### PR DESCRIPTION
**Pull request by Nayira-Sanchez**
Wording amended for the Overview section

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me, just a minor question about including some extra context in the wording 🙂

**Comment by Nayira-Sanchez** [comment]
Comment discussed with BA and further changes added
/azp run

File: specification/e-referrals-service-api.yaml [hunk 1]
```diff
@@ -58,8 +58,8 @@ info:
...
+    | Application-restricted, unattended access | [Signed JWT](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)    | [A004](#api-Default-a004-retrieve-reference-data), [A005](#api-Default-a005-retrieve-referral-request), [A006](#api-Default-a006-retrieve-attachment), [A007](#api-Default-a007-retrieve-clinical-information), [A024](#api-Default-a024-retrieve-advice-and-guidance), [A025](#api-Default-a025-retrieve-advice-and-guidance-conversation) and [A043](#api-Default-a043-retrieve-advice-and-guidance-overview-pdf) | [in production, beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses) |
+    | Healthcare worker, user-restricted access | [CIS2](https://digital.nhs.uk/services/identity-and-access-management/nhs-care-identity-service-2/care-identity-authentication/ways-to-authenticate-with-nhs-cis2)  | All [Endpoints](#api-endpoints) | [in production, beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses) |
```
Comment (nhsd-jack-wainwright) [inline specification/e-referrals-service-api.yaml:62]: Should the I in `in` be captialised here?

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂

**Review by stuandgray — APPROVED** [review]

**Pull request merged**